bundles/org.openhab.binding.shelly/README.md

```diff
@@ -0,0 +1,437 @@
+|          |last_update  |String   |yes      |Timestamp of the last update (values read by the binding)              |
+|          |event        |Trigger  |yes      |Trigger channel, receives JSON formatted event information             |
+|battery   |batteryLevel |Number   |yes      |Battery Level in %                                                     |
+|          |voltage      |Number   |yes      |Voltage of the battery                                                 |
+|          |lowBattery   |Switch   |yes      |Low battery alert (< 20%)                                              |
+
+
+### Shelly Sense
+
+|Group     |Channel      |Type     |read-only|Description                                                            |
+|----------|-------------|---------|---------|-----------------------------------------------------------------------|
+|control   |key          |String   |r/w      |Send a IR key to the sense. There a 3 different types supported        |
+|          |             |         |         |Stored key: send the key code defined by the App , e.g. 123_1_up       |
+|          |             |         |         |Pronto hex: send a Pronto Code in ex format, e.g. 0000 006C 0022 ...   |
+|          |             |         |         |Pronto base64: in base64 format, will be send 1:1 to the Sense         |
+|          |motionTime   |Number   |r/w      |Define the number of seconds when the Sense should report motion       |
+|          |motionLED    |Switch   |r/w      |Control the motion LED: ON when motion is detected or OFF              |
+|          |charger      |Switch   |yes      |ON: charger connected, OFF: charger not connected.                     |
+|sensors   |temperature  |Number   |yes      |Temperature in °C                                                      |
+|          |humidity     |Number   |yes      |Relative humidity in %                                                 |
+|          |lux          |Number   |yes      |Brightness in Lux                                                      |
+|          |motion       |Switch   |yes      |ON: Motion detected, OFF: No motion (check also motionTimer)           |
+|          |last_update  |String   |yes      |Timestamp of the last update (values read by the binding)              |
+|          |event        |Trigger  |yes      |Trigger channel, receives JSON formatted event information             |
+|battery   |batteryLevel |Number   |yes      |Battery Level in %                                                     |
+|          |batteryAlert |Switch   |yes      |Low battery alert                                                      |
+
+
+## Full Example
+
+Note: PaperUI is recommended, if you want to use text files make sure to replace the thing id from you channel definition 
+
+Replace roller:XXXXXn with the last 6 digits of the Shelly device's MAC address.
```

Comment:
Why? The Thing ID can be chosen by the user, so calling it "1" should just be fine.

Reply:
removed